src/main/kotlin/ch/hevs/cloudio/cloud/security/BrokerSecurityService.kt

```diff
@@ -122,7 +125,7 @@ class BrokerSecurityService(private val userDetailsService: CloudioUserDetailsSe
                     }
                     else -> try {
                         (userDetailsService.loadUserByUsername(id) as CloudioUserDetails).let { userDetails ->
-                            if (modelIdentifier.count() == 0 || modelIdentifier.wildcard) {
+                            if (modelIdentifier.count() == 1 && modelIdentifier[0] == "#") {
```

Comment:
This might break access control for HTTP access...

Reply:
This line only concern the mqtt authorization as it is in the BrokerSecurityService.

Reply:
Then it is fine, my bad.